### ISSUE TITLE
[bug] broken -work-dir param

### ISSUE THREAD

**Issue by simone-pignotti** [bactopia/bactopia#165]
**Description**
The `-work-dir` command-line parameter looks broken.

**Steps to Reproduce**
```
$ bactopia -work-dir test
Can only specify option -w once. -- Check the available commands and options and syntax with 'help'
```

**Expected Behavior**
The default `WORK_DIR` variable of the `bactopia` shell script should be overwritten and passed to the `nextcloud run` command.

**Execution Environment**

- Bactopia Version: `1.5.3`
- OS: Debian 10
- Environment: conda

**Additional Information**
I tried to add a `WORK_DIR=""` to make sure an existing env variable would not be passed on, but that doesn't help.

Thanks for this great pipeline!

**Comment by rpetit3** [comment]
I'm going to dig into this. 

This works
```
bactopia --accession SRX1390608 -work-dir test2 --condadir /data/storage/bactopia-ci/bactopia-envs
```

But like you showed, this does not:
```
bactopia -work-dir test --condadir /data/storage/bactopia-ci/bactopia-envs
Can only specify option -w once. -- Check the available commands and options and syntax with 'help'
```

If I run it directly, it works
```
nextflow run ~/miniconda3/envs/bactopia/share/bactopia-1.5.x/main.nf -w test
N E X T F L O W  ~  version 20.10.0
Launching `/local/home/rpetit/miniconda3/envs/bactopia/share/bactopia-1.5.x/main.nf` [spontaneous_mcclintock] - revision: 403ec4cd2e
One or more required parameters are missing, please check and try again.

Required Parameters:
    ### For Procesessing Multiple Samples
    --fastqs STR            An input file containing the sample name and
                                absolute paths to FASTQ/FASTAs to process

    ### For Processing A Single Sample
    --R1 STR                First set of reads for paired end in compressed (gzip)
                                FASTQ format

    --R2 STR                Second set of reads for paired end in compressed (gzip)
                                FASTQ format

    --SE STR                Single end set of reads in compressed (gzip) FASTQ format

    --hybrid                The SE should be treated as long reads for hybrid assembly.

    --sample STR            The name of the input sequences

    ### For Downloading from SRA/ENA or NCBI Assembly
    **Note: Assemblies will have error free Illumina reads simulated for processing.**
    --accessions            An input file containing ENA/SRA Experiment accessions or
                                NCBI Assembly accessions to be processed

    --accession             A single ENA/SRA Experiment accession or NCBI Assembly accession
                                to be processed

    ### For Processing an Assembly
    **Note: The assembly will have error free Illumina reads simulated for processing.**
    --assembly STR          A assembled genome in compressed FASTA format.
Cannot continue, please see --help for more information
```

I'm guessing my bash logic is off.

**Comment by simone-pignotti** [comment]
Thanks for the quick reply, yes I am pretty sure it's a bash issue as well, but can't figure out why it still happens after manually setting WORK_DIR=""...let me know if you need more info!

**Comment by rpetit3** [comment]
OK think I've figured it out, definitely the bash logic. I'll push a fix soon.

Thanks for finding this!

**Comment by simone-pignotti** [comment]
Great, thank you! :)

**Comment by rpetit3** [comment]
Alright submitted a fix https://github.com/bactopia/bactopia/commit/314b63c644512b7f8ebba0841557584532012d45 I'll push it in the next update.

For the time being, you can use `-w` or `-work-dir` as long as its not the first parameters:

```
OK   - bactopia --cpus 4 -w test
FAIL - bactopia -w test --cpus 4
FAIL - bactopia -w test
```

**Comment by rpetit3** [comment]
Hi @simone-pignotti 

I released v1.5.4, so it should be available on Bioconda soon (6-12hours, maybe less). I'm going to close this for now, please feel free to reopen if the issue persists.

Thank you very much for your help!
Robert
